### PR TITLE
add contributing.md

### PR DESCRIPTION
**Pull request by stanleygomes**
# PR Details

## Description

Give a good explanation about this PR, including about the files you've changed.

## Related Issue 

Issue Title ([PR_ID](http://github.com/link_for_issue))

## Steps to Reproduce

`Write your steps here:`

1. Sample
2. Of
3. Steps

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by hugonasciutti** [comment]
Boa @stanleygomes.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```
Comment (fredericofigueira) [inline CONTRIBUTING.md]: You can remove the second please word, like this:

"Please note we have a code of conduct, follow it in all your interactions with the proejct."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance and religion.
```
Comment (fredericofigueira) [inline CONTRIBUTING.md]: On the phrase below I think it's better to change the 'and ' word to 'or' 
"...personal appearance and religion."

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+### Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
```
Comment (fredericofigueira) [inline CONTRIBUTING.md]: I think the right writing is includes, in plural

File: CONTRIBUTING.md (hunk 2 above)
Comment (vilaboim) [reply CONTRIBUTING.md]: I disagree.
We want a harassment-free experience regardless of all of those aspects, not one or another.

File: CONTRIBUTING.md (hunk 3 above)
Comment (vilaboim) [reply CONTRIBUTING.md]: `include` is a verb, `s` is necessary after `he`, `she` and `it`

File: CONTRIBUTING.md (hunk 2 above)
Comment (fredericofigueira) [reply CONTRIBUTING.md]: I agree with you, it can be wrong understood with 'or' word, my bad

Comment (stanleygomes) [reply CONTRIBUTING.md]: What???

Comment (vilaboim) [reply CONTRIBUTING.md]: It is resolved.

**Pull request merged**